_resources/port1.0/group/python-1.0.tcl

```diff
@@ -337,7 +343,12 @@ options python.branch python.prefix python.bin python.lib python.libdir \
         python.test_framework python.add_dependencies
 # for pythonXY, python.branch is X.Y, for pythonXYZ, it's X.YZ
 default python.branch   {[string index ${python.version} 0].[string range ${python.version} 1 end]}
-default python.prefix   {${frameworks_dir}/Python.framework/Versions/${python.branch}}
+
+# Need a normalized python.prefix, in case frameworks_dir has a sym link.
+set     frameworks_abs   [file normalize ${frameworks_dir}]
+default python.prefix    {${frameworks_dir}/Python.framework/Versions/${python.branch}}
+default python.absprefix {${frameworks_abs}/Python.framework/Versions/${python.branch}}
+
```

Comment:
is there a reason why one cannot always use a "normalized" version? In other words why can one not just do `default python.prefix   {[file normalize ${frameworks_dir}/Python.framework/Versions/${python.branch}]}` and keep the rest of the PortGroup as is - I see no real benefit of introducing additional variables and or the `ui_msg` statements

Reply:
@reneeotten you might be right.  I did not want to risk perturbing the other uses of `python.prefix`.  Those seems to work fine the way they are.  That would be doubly true for `frameworks_dir` from base.  I thought a new variable would be the most conservative approach, targeting this single purpose.

The extra print statements are merely diagnostics, compensating for Macports build system frequently hiding things I want to know.  I could take those out, or switch to `ui_debug`, or something

With that, I will gladly accept your preference on these things.  You understand portgroups very well, I am barely getting started.

Reply:
these were just my thoughts after a quick read. I would like to have @jmroot his feedback as he wrote most of the "python" PortGroup and has a much better understanding of it and possible implications of this change than most of us.